spec/accountSpec.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var Account = require('../src/account').Account
+var validateDate = require('../src/account').validateDate
+var convertStringToDate = require('../src/transaction').convertStringToDate
+
+describe('Account', function () {
+  var account, date, transaction, dateConverted;
+  beforeEach(function () {
+    date = '10/01/2012'
+    dateConverted = convertStringToDate(date)
+    account = new Account();
+  });
+
+  describe('is initialized', function () {
+    it('with an empty array of transactions', function () {
+      expect(account.transactions).not.toBe(null);
+    });
+  });
+
+  describe('#recordTransaction', function () {
+    it('records a new transaction in the account', function () {
+      account.recordTransaction(100, date);
+      transaction = account.transactions.pop();
+      expect(transaction.date.getTime()).toBe(dateConverted.getTime());
```

Comment:
This line is checking the code against itself.  It checks that the `transaction.date` is the same as the result of taking the test date and calling the `convertStringToDate` function on it.  It's fine for a test to check that a method has been called, but that involves just checking the method was called, not calling it and checking its return value.  Let's talk about this more if it's not clear what I'm saying!